packages/core/src/Renderer.ts

```diff
@@ -120,12 +140,12 @@ export class Renderer extends AbstractRenderer
          */
         this.runners = {
             destroy: new Runner('destroy'),
-            contextChange: new Runner('contextChange', 1),
+            contextChange: new Runner('contextChange'),
             reset: new Runner('reset'),
             update: new Runner('update'),
             postrender: new Runner('postrender'),
             prerender: new Runner('prerender'),
-            resize: new Runner('resize', 2),
+            resize: new Runner('resize'),
         };
 
         /**
```

Comment:
The comments from line 159-162 can be removed. It was added temporarily for the display object conversion